itests/util/src/main/java/org/apache/hadoop/hive/hbase/HBaseTestSetup.java

```diff
@@ -106,6 +106,8 @@ private void setUpFixtures(HiveConf conf) throws Exception {
         "org.apache.hadoop.hbase.shaded.");
 
     Configuration hbaseConf = HBaseConfiguration.create(conf);
+    // A workaround for HBASE-28908
+    hbaseConf.set("hbase.wal.provider", "filesystem");
```

Comment:
can you create a Hive ticket to upgrade HBase to 2.7.0. & link the mentioned ticket + this line to be removed + HBASE-28939

Reply:
Created the ticket HIVE-28740.